packages-next/keystone/src/admin-ui/static/next.config.js

```diff
@@ -1,3 +1,3 @@
 // this file is copied somewhere else so this lint rule is incorrect
```

Comment:
This and the tsconfig.json in the same directory need to the at the root of the package (outside of src)

Reply:
When i said root of the package, I meant in the static directory at the root of the repo